frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveImage/index.tsx

```diff
@@ -66,9 +71,19 @@ export default function SaveImage({
   }, [isOpen]);
 
   const handleDownload = useCallback(async () => {
+    setIsDownloading(true);
     try {
       const heatmapNode = document.getElementById("view") as HTMLCanvasElement;
+
+      //(ashin): #3569 Get scrollTop to go back to place after downloading image
+      let heatmapContainer = document.getElementById(
+        HEATMAP_CONTAINER_ID
+      ) as HTMLCanvasElement;
+      heatmapContainerScrollTop = heatmapContainer?.scrollTop;
+
+      // Adding this class causes the y-axis scrolling to jump but is required for image download
```

Comment:
Thanks for adding the comment! Saved me time to figure out what was causing the jump lol